### PR TITLE
🌐 Add Bengali translations for `docs/bn/docs/python-types.md`

### PR DESCRIPTION
**Pull request by imtiaz101325**
This pull request adds Bengali translations for the `docs/bn/docs/python-types.md` file.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9d15da25fd6a23ddd3790d8d6e26c31e0a67e4ff at: https://baa0dc1c.fastapitiangolo.pages.dev

**Comment by codespearhead** [comment]
Do you happen to know three Bengali developers that you can ask a [PR review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from? [This](https://github.com/tiangolo/fastapi/pulls?q=is%3Apr+bengali) might help.

**Review by shabab477 — APPROVED** [review]
Hi Bengali developer here, LGTM !

File: docs/bn/docs/img/python-types/image01.png
Comment (tiangolo) [inline docs/bn/docs/img/python-types/image01.png:1]: Please remove these images, using the original URLs from the original doc in English should work fine. :nerd_face:

**Review by tiangolo — CHANGES_REQUESTED** [review]
Please remove the image files. :nerd_face:

**Comment by tiangolo** [comment]
📝 Docs preview for commit d1eb957c20125f04ac17c9b46d6ae46afacfd291 at: https://37cc6e39.fastapitiangolo.pages.dev

File: docs/bn/docs/img/python-types/image01.png
Comment (imtiaz101325) [reply docs/bn/docs/img/python-types/image01.png:1]: Ah! Did not know. It wasn't working locally. Updated it!

Comment (tiangolo) [reply docs/bn/docs/img/python-types/image01.png:1]: Great, thanks!

**Review by tiangolo — APPROVED** [review]

**Pull request merged**